consensus/buildtest.go

```diff
@@ -32,7 +31,7 @@ var (
 
 	CoinbaseAugment = new(big.Int).Lsh(big.NewInt(1), 80)
```

Comment:
it seems though that it  was meant to be used this way. I  think nil is a pretty common part of using Int.Exp

I'd much rather have exp(10, 24, nil) than Set("10000000howmany?000")